numpy/random/bit_generator.pyx

```diff
@@ -116,7 +116,7 @@ def _coerce_to_uint32_array(x):
     Examples
     --------
     >>> import numpy as np
-    >>> from np.random.bit_generator import _coerce_to_uint32_array
+    >>> from numpy.random.bit_generator import _coerce_to_uint32_array
```

Comment:
Why is this needed? The line above should make the old code work ...

Reply:
> Why is this needed?

I'm not sure which part of the change "this" refers to.  The import statement is needed so we can use `_coerce_to_uint32_array` without giving the fully qualified name.  The change from `np` to `numpy` is needed because a Python variable can't be used as part of the module in an import statement.

Reply:
I don't think the `import numpy as np` is needed for examples, but that is extraneous to this PR.